tests/test_GuideRate.cpp

```diff
@@ -503,23 +503,23 @@ BOOST_AUTO_TEST_CASE(P_Third_df09)
 
     // GR_{oil}
     {
-        const auto grval = cse.gr.get("P", Opm::Well::GuideRateTarget::OIL, { orat, grat, wrat });
+        const auto grval = cse.gr.getProductionGroup("P", Opm::Group::GuideRateProdTarget::OIL, { orat, grat, wrat });
 
         const auto expect = 0.9*expect_gr_oil_3 + 0.9*0.1*expect_gr_oil_2 + 0.1*0.1*expect_gr_oil_1;
         BOOST_CHECK_CLOSE(grval, expect, 1.0e-5);
     }
 
     // GR_{gas}
     {
-        const auto grval = cse.gr.get("P", Opm::Well::GuideRateTarget::GAS, { orat, grat, wrat });
+        const auto grval = cse.gr.getProductionGroup("P", Opm::Group::GuideRateProdTarget::GAS, { orat, grat, wrat });
 
         const auto expect = (grat / orat) * (0.9*expect_gr_oil_3 + 0.9*0.1*expect_gr_oil_2 + 0.1*0.1*expect_gr_oil_1);
         BOOST_CHECK_CLOSE(grval, expect, 1.0e-5);
     }
 
     // GR_{water}
     {
-        const auto grval = cse.gr.get("P", Opm::Well::GuideRateTarget::WAT, { orat, grat, wrat });
+        const auto grval = cse.gr.getProductionGroup("P", Opm::Group::GuideRateProdTarget::WAT, { orat, grat, wrat });
 
```

Comment:
This is an example of why I changed the name of the methods and wanted to avoid overloading. "P" is a group but in the original code it got a guiderate target applicable for the wells only. A hard but to notice. I had some similar issues in the wellmodel in opm-simulators.

Reply:
OK - I see.

Reply:
I can go back to the "overloading" if you prefer that. Your choice.

Reply:
@joakim-hove Any conclusions on what you prefer here?

Reply:
I prefer overøoad and short names.

Reply:
Ok. Give me some minutes  and I will update accordingly.